src/components/pages/AccommodationEditPage.jsx

```diff
@@ -60,6 +69,23 @@ class AccommodationEditPage extends React.Component {
         );
     };
 
+    fetchAccommodationBefore = () => {
+        this.setState({ status: FETCHING });
+    };
+    fetchAccommodationSuccess = ({ data }) => {
+        this.setState({
+            status: FETCH_DONE,
+            fetchFinished: true,
```

Comment:
The purpose of `status` field, IIRC, was, amongst others, indicate that fetch is Finished. In other words, there is some  redundancy here (but I might be wrong, feel free to keep it if required)